tox.ini

```diff
@@ -1,9 +1,24 @@
 [tox]
-envlist = py27,py35,py36,py37,py38,pypy,pypy3
+envlist = py27,py35,py36,py37,py38,py39,pypy,pypy3,coverage
 
 [testenv]
-deps =
-    setuptools >= 17.1
-    mock <= 3.0.5; python_version == "2.7"
+# usedevolop is an easy way to fix multiple paths
+# not matching in the coverage output.
+usedevelop = true
```

Comment:
Since you already use `coverage combine`, the other way of fixing multiple paths is adding these five lines to `.coveragerc`:

```
[paths]
source =
    src/
    .tox/*/lib/python*/site-packages/
    .tox/pypy*/site-packages/
```